src/mainboard/sunxi/nezha/bt0/src/main.rs

```diff
@@ -571,6 +602,33 @@ extern "C" fn main() {
         // flash is freed when it goes out of scope
     }
 
+    #[cfg(feature = "mmc")]
+    {
+        let sdc0_d1 = gpio.portf.pf0.into_function_2();
+        let sdc0_d0 = gpio.portf.pf1.into_function_2();
+        let sdc0_clk = gpio.portf.pf2.into_function_2();
+        let sdc0_cmd = gpio.portf.pf3.into_function_2();
+        let sdc0_d3 = gpio.portf.pf4.into_function_2();
+        let sdc0_d2 = gpio.portf.pf5.into_function_2();
+        let sdc0_pins = (sdc0_d1, sdc0_d0, sdc0_clk, sdc0_cmd, sdc0_d3, sdc0_d2);
+
+        let mut smhc = Smhc::new(p.SMHC0, sdc0_pins, 200_000_000.hz(), &clocks);
+
+        // Mandatory 8K SPL offset on sdcard
+        let skip = 8192;
+        // SPL is already loaded (by BROM), so skip it
+        let skip = skip + (0x1 << 15); // 32K, the size of SPL/boot0
+        load(skip, ORE_ADDR, ORE_SIZE, &mut smhc).ok();
+
+        // Linux is behind oreboot + dtfs
+        let skip = skip + ORE_SIZE + DTF_SIZE;
+        load(skip, LIN_ADDR, LIN_SIZE, &mut smhc).ok();
+
+        // DTB is behind Linux
+        let skip = skip + LIN_SIZE;
+        load(skip, DTB_ADDR, DTB_SIZE, &mut smhc).ok();
```

Comment:
what kind of diagnostic do we get when things fail in load()?

Reply:
I wonder what the right behavior is when loading fails.
Retrying, stopping or just continuing anyway.
At the very least I'll add a `println!` for the error, I'm doing that locally as well.